disperser/cmd/controller/flags/flags.go

```diff
@@ -0,0 +1,182 @@
+		Name:     common.PrefixFlag(FlagPrefix, "encoding-store-timeout"),
+		Usage:    "Timeout for interacting with blob store",
+		Required: false,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "ENCODING_STORE_TIMEOUT"),
+		Value:    15 * time.Second,
+	}
+	NumEncodingRetriesFlag = cli.IntFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "num-encoding-retries"),
+		Usage:    "Number of retries for encoding requests",
+		Required: false,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "NUM_ENCODING_RETRIES"),
+		Value:    3,
+	}
+	NumRelayAssignmentFlag = cli.IntFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "num-relay-assignment"),
+		Usage:    "Number of relays to assign to each encoding request",
+		Required: false,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "NUM_RELAY_ASSIGNMENT"),
+		Value:    2,
+	}
+	NumConcurrentEncodingRequestsFlag = cli.IntFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "num-concurrent-encoding-requests"),
+		Usage:    "Number of concurrent encoding requests",
+		Required: false,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "NUM_CONCURRENT_ENCODING_REQUESTS"),
+		Value:    250,
+	}
+
+	// Dispatcher Flags
+	DispatcherPullIntervalFlag = cli.DurationFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "dispatcher-pull-interval"),
+		Usage:    "Interval at which to pull from the queue",
+		Required: true,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "DISPATCHER_PULL_INTERVAL"),
+	}
+	NodeRequestTimeoutFlag = cli.DurationFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "node-request-timeout"),
+		Usage:    "Timeout for node requests",
+		Required: true,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "NODE_REQUEST_TIMEOUT"),
+	}
+	NumConnectionsToNodesFlag = cli.IntFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "num-connections-to-nodes"),
+		Usage:    "Max number of connections to nodes",
+		Required: true,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "NUM_CONNECTIONS_TO_NODES"),
+	}
+	FinalizationBlockDelayFlag = cli.Uint64Flag{
+		Name:     common.PrefixFlag(FlagPrefix, "finalization-block-delay"),
+		Usage:    "Number of blocks to wait before finalizing",
+		Required: false,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "FINALIZATION_BLOCK_DELAY"),
+		Value:    75,
+	}
+	NumRequestRetriesFlag = cli.IntFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "num-request-retries"),
+		Usage:    "Number of retries for node requests",
+		Required: false,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "NUM_REQUEST_RETRIES"),
+		Value:    3,
+	}
+	NumConcurrentDispersalRequestsFlag = cli.IntFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "num-concurrent-dispersal-requests"),
+		Usage:    "Number of concurrent dispersal requests",
+		Required: false,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "NUM_CONCURRENT_DISPERSAL_REQUESTS"),
+		Value:    600,
+	}
+	NodeClientCacheSizeFlag = cli.IntFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "node-client-cache-size"),
+		Usage:    "Size of the node client cache",
```

Comment:
What's the unit of this?

Reply:
Number of entries. Updated the name accordingly